README.md

```diff
@@ -9,6 +9,115 @@
+
+```swift
+// Animating implicitly with UIView APIs
+UIView.animate(withDuration: 1.0, animations: {
+  view.alpha = 0.5
+})
+
+// Equivalent MotionAnimator API
+MotionAnimator.animate(withDuration: 1.0, animations: {
+  view.alpha = 0.5
+})
+```
+
+But the MotionAnimator allows you to animate more properties — and on more iOS versions:
+
+```swift
+UIView.animate(withDuration: 1.0, animations: {
+  view.layer.cornerRadius = 10 // Only works on iOS 11 and up
+})
+
+MotionAnimator.animate(withDuration: 1.0, animations: {
+  view.layer.cornerRadius = 10 // Works on iOS 8 and up
+})
+```
+
+MotionAnimator makes use of the [MotionInterchange](https://github.com/material-motion/motion-interchange-objc), a standardized format for representing animation traits. This makes it possible to tweak the traits of an animation without rewriting the code that ultimately creates the animation, useful for building tweaking tools and making motion "stylesheets".
+
+```swift
+// Want to change a trait of your animation? You'll need to use a different function altogether
+// to do so:
+UIView.animate(withDuration: 1.0, animations: {
+  view.alpha = 0.5
+})
+UIView.animate(withDuration: 1.0, delay: 0.5, options: [], animations: {
+  view.alpha = 0.5
+}, completion: nil)
+
+// But with the MotionInterchange, you can create and manipulate the traits of an animation
+// separately from its execution.
+let traits = MDMAnimationTraits(duration: 1.0)
+traits.delay = 0.5
+
+let animator = MotionAnimator()
+animator.animate(with: traits, animations: {
+  view.alpha = 0.5
+})
+```
+
+The MotionAnimator can also be used to replace explicit Core Animation code with additive explicit animations:
+
+```swift
+// Animating expicitly with Core Animation APIs
+let animation = CABasicAnimation(keyPath: "cornerRadius")
+animation.fromValue = 0
+animation.toValue = 10
+animation.duration = 1.0
+// Note: this animation won't be additive unless we configure it accordingly.
+view.layer.add(animation, forKey: animation.keyPath)
+
+// Equivalent implicit MotionAnimator API. cornerRadius will be animated additively by default.
+view.layer.cornerRadius = 0
+MotionAnimator.animate(withDuration: 1, animations: {
+  view.layer.cornerRadius = 10
+})
+
+// Equivalent explicit MotionAnimator API
+// Note that this API will also set the final animation value to the layer's model layer, similar
+// to how implicit animations work, and unlike the explicit pure Core Animation implementation
+// above.
+let animator = MotionAnimator()
+animator.animate(with: MDMAnimationTraits(duration: 1.0),
+                 between: [0, 10],
+                 layer: view.layer,
+                 keyPath: .cornerRadius)
+```
+
+Springs on iOS require an initial velocity that's normalized by the displacement of the animation. MotionAnimator calculates this for you so that you can directly provide gesture recognizer velocity values:
+
+```swift
+// Common variables
+let gestureYVelocity = ...
```

Comment:
Is the ellipsis a Swift structure?

Reply:
Nope - just didn't want to write the actual code here. Will add it.